css/selectors.json

```diff
@@ -342,14 +342,14 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:link"
   },
-  ":matches": {
-    "syntax": ":matches( <selector># )",
+  ":is": {
+    "syntax": ":is( <complex-selector># )",
     "groups": [
       "Pseudo-classes",
       "Selectors"
     ],
     "status": "standard",
```

Comment:
Also, this is probably `experimental` now.

Reply:
Ack, forgot about that, thanks! I did set it as experimental in BCD.